File: music-player-lib/src/main/java/com/music/player/lib/view/MusicMarqueeTextView.java
package com.music.player.lib.view;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.Nullable;
import androidx.appcompat.widget.AppCompatTextView;

/**
 * TinyHung@Outlook.com
 * 2017/11/27.
 */

public class MusicMarqueeTextView extends AppCompatTextView {

    public MusicMarqueeTextView(Context context) {
        super(context);
    }

    public MusicMarqueeTextView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public MusicMarqueeTextView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public boolean isFocused() {
        return true;
    }
}

File: music-exoplayer-lib/src/main/java/com/rm/music_exoplayer_lib/view/MusicMarqueeTextView.kt
package com.rm.music_exoplayer_lib.view

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView

/**
 *
 * @des:
 * @data: 9/2/20 10:27 AM
 * @Version: 1.0.0
 */
class MusicMarqueeTextView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet,
    defStyle: Int = 0
) : AppCompatTextView(context, attrs, defStyle) {

    override fun isFocused(): Boolean {
        return true
    }
}